portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,20 @@ function addRandomFunFacts() {
   const funFactContainer = document.getElementById('funFacts-container');
   funFactContainer.innerText = funFact;
 }
+
+async function showRandomGreets() {
+    console.log("testing testing");
```

Comment:
Nit: let's remove the console log before merging into main

Reply:
Thanks for catching that! Completely forgot I left that there